libs/SalesforceSDKCore/SalesforceSDKCoreTests/PushNotificationManagerTests.swift

```diff
@@ -187,13 +187,13 @@ class PushNotificationManagerTests: XCTestCase {
                             "actions": [
                                 {
                                     "name": "approve",
-                                    "identifier": "approval_req__approve",
+                                    "actionKey": "approval_req__approve",
```

Comment:
updating the mock data to reflect what I'm getting back from a live server call.